docs/howToRun.md

```diff
@@ -1,7 +1,20 @@
 # How to run Flinkfood
 
-The next few steps explain how to run the Job that makes the first letter of a user name uppercase:
+This file is intended to give you an introduction on how to run the Flinkfood application. There is an easy way. And there is an hard way, which contains more details on how things are setup.
 
```

Comment:
I think it would be nice to add a pre-requisites subsession here on what the person has to have installed on their machine in order to run (i.e. docker), what do you think? Just to make the document more complete